models/odos/optimism/odos_v2_optimism_trades.sql

```diff
@@ -0,0 +1,102 @@
+{{ config(
+    alias = 'trades'
+    ,schema = 'odos_v2_optimism'
```

Comment:
Hi @ARDev097,
have you added `odos_v2_optimism.trades` into `odos_optimism.trades` roll-up spell?

Reply:
Hello @Hosuke ,
I have added `odos_v2_optimism.trades` into `odos_optimism.trades`.